### DISCUSSION TITLE
Can i automatically re-onboard all discovered repositories?

### DISCUSSION THREAD

**Discussion by danielhammerl** [renovatebot/renovate#13067] [Request Help] [answered]
Hi guys, so i have round about 70 git repositories which where all autodiscovered and auto-onboarded.
I know detected an error in my onboard configuration and dont want to open 75 PullRequests for renovate to re-onboard those repositories. Is there any way to force renovate to re-onboard every repository found?

**Comment by rarkins** [comment, marked as answer]
I don't think so, but would also like to understand what you mean by "re-onboard". Do you mean the original and wrong onboarding config was already merged into default branch by all repos? As in, they all have a config already?

**Reply by danielhammerl** [reply]
yes exactly, every repo has the wrong config and i would like to now if there is a way to automatically replace the wrong config with the updated one.

**Reply by rarkins** [reply]
No, there is no way to do that using Renovate. You'd need something like https://github.com/octoherd/octoherd

**Reply by danielhammerl** [reply]
alright, thank you very much
